components/Action/src/ActionDate.tsx

```diff
@@ -10,15 +8,15 @@ interface Props {
   dateTime: string;
   now?: string;
   relative?: boolean;
-  locale?: Locale;
-  labels: FormatDateLabels;
+  locale?: string;
+  labels?: FormatDateLabels;
 }
 
 export const ActionDate = ({
   dateTime,
   now = new Date().toISOString(),
   relative = false,
-  locale = nl,
+  locale = 'nl-NL',
```

Comment:
Moeten we hier dan ook de navigator toevoegen anders is dit de default of deze default undefined maken.

Reply:
Huh, ik ben de changes vergeten te committen haha. Had de default undefined gemaakt.